### ISSUE TITLE
Notes on CSS

### ISSUE THREAD

**Issue by SidTheEngineer** [MarcWk/f1-3-c2p1-colmar-academy#2]
I've highlighted a few areas for improvement below:

**Desktop width styling**

Although the mobile width version of the site looks perfect, our elements at desktop width are a bit jumbled. In particular, there are elements in the Banner and Information sections that are a bit too large, and end up flowing into the other divs of the site. See if you can adjust the sizing at desktop width for these sections and utilize alignment to make sure these don't overflow into other elements.

Likewise, the "Start Learning" section of the site does not utilize space appropriately. See if you can create more columns for elements to take up at desktop width in this situation (instead of the two columns that are used).

**Comment by MarcWk** [comment]
You're right. Somehow I made it on 70% screen width in the Chrome settings, then it is ok. When opening it in 100%, it's mixed up. 
I guess it's because I thought 1 rem is 8px, probably it is 16px? So I made 32px 4 rem instead of 2 rem, etc
